phosphor-instrument-jigsaw/src/main/java/edu/columbia/cs/psl/jigsaw/phosphor/instrumenter/StandaloneJVMInstrumenter.java

```diff
@@ -24,7 +24,7 @@ public static void main(String[] args) {
     private static Properties toProperties(CommandLine line){
         Properties ret = new Properties();
         for(Option opt : line.getOptions()){
-            String name = opt.getArgName();
```

Comment:
What is this change for?

Reply:
This change fixes a NullPointerException while using `StandaloneJVMInstrumenter` with configuration options. The option names are stored in `opts` not in `argName`.